src/ess/livedata/dashboard/data_subscriber.py

```diff
@@ -118,7 +139,13 @@ def trigger(self, store: dict[Key, Any]) -> None:
         """
         data = {key: store[key] for key in self.keys if key in store}
         assembled_data = self._assembler.assemble(data)
-        self._pipe.send(assembled_data)
+
+        if self._pipe is None:
+            # First trigger - create pipe with correctly extracted data
+            self._pipe = self._pipe_factory(assembled_data)
```

Comment:
Should we also `pipe.send(...)` after the pipe was created?

Reply:
Creating the pipe (this is `hv.streams.Pipe`) sends the data passed in this init call. We don't want to send it twice. There is no way to init without data, that is way we had to move it into the subscriber and create on first use.